src/betterproto/__init__.py

```diff
@@ -818,9 +818,10 @@ def to_dict(
         """
         output: Dict[str, Any] = {}
         field_types = self._type_hints()
-        for field_name, meta in self._betterproto.meta_by_field_name.items():
+        proto_meta = self._betterproto
```

Comment:
I don't think this it worth it here. I'm guessing the point is that for the cost of adding an assignment in scope, you save a lookup step for the value later?

However the perf value of this here is certainly below the level of noise for even a contrived test case, and I feel it makes the code less clear by adding a level of indirection without really clarifying anything in the process.

Reply:
Ok